### ISSUE TITLE
Rotate Actor around pivot point. 

### ISSUE THREAD

**Issue by fortmarek** [JohnSundell/ImagineEngine#164]
I would like to rotate `Actor` with `UIImage` to rotate around a defined pivot point but I have not been able to find how to achieve this. 

When I tried to find a solution, the best answer was this: https://stackoverflow.com/questions/27658454/swift-rotating-an-image-in-an-uiimageview-defined-in-the-main-storyboard

What I am trying to achieve: 

![demo](https://raw.githubusercontent.com/fortmarek/issue_images/master/Apple%20Watch%2038mm_0-20.gif)

**Comment by Alex88WH** [comment]
Hey there @fortmarek I figured that out
![jan-31-2018 18-09-22](https://user-images.githubusercontent.com/8467996/35645365-b2036268-06b2-11e8-8624-bb7513bbd9c8.gif)
Check out CALayer's anchorPoint function. Actor has a private layer property. Just set a new value (create a public variable 😁 ).

**Comment by fortmarek** [comment]
Thanks! Maybe it would be great if this was made available as a public property, @JohnSundell? 😄

**Comment by JohnSundell** [comment]
@fortmarek Yeah sounds great, feel free to implement it as a public property on `Actor` and send a PR👍 Just like `opacity`, `cornerRadius` etc it can just forward the value directly to the layer 🙂

**Comment by JohnSundell** [comment]
Just one thing to keep in mind is that this property may behave a bit differently on macOS compared to iOS+tvOS, due to coordinate system differences. Just something to test for as this is being implemented 🙂
